adapt/context.py

```diff
@@ -8,11 +8,19 @@ class ContextManagerFrame(object):
     Manages entities and context for a single frame of conversation.
     Provides simple equality querying.
     """
-    def __init__(self, entities=[], metadata={}):
-        self.entities = entities
-        self.metadata = metadata
+    def __init__(self, entities=None, metadata=None):
+        if entities is None:
```

Comment:
This worked when running tests except for in the case of:
`    def __init__(self, trie, tokenizer, regex_entities=None, max_tokens=20):
        self.trie = trie
        self.tokenizer = tokenizer
        self.max_tokens = max_tokens
        if regex_entities is None:
            self.regex_entities = []
        else:
            self.regex_entities = regex_entities`

In entity_tagger.py
Any ideas why this would be?